WTWSMS/common/landed_titles/old_duchies.txt

```diff
@@ -1255,8 +1267,8 @@ d_trebizond = {
 }
 d_armeniacon = {
 	color={ 255 0 167 }
-	color2={ 255 255 20 }
-	
+	color2={ 255 255 20 }	
+
 	capital = 738 # Amisos
 	allow = {
 		always = no
```

Comment:
So you mean creation by Decision? What the heck is a "dynamic de jure"? Are you hinting that these titles will be possible?